### PR TITLE
updated README file with build instruction and basic usage information

### PR DESCRIPTION
**Pull request by Nikhil-Kumar-ML**
Checklist:

- [x] I have read [contribution guidelines](https://github.com/maplelabs/github-audit/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] I have tested these changes locally on my machine.

##### SUMMARY

Updated README file with basic build and usage instruction

### CODE REVIEW

**Comment by Aswin-Bhupalam-ML** [comment]
Please go through https://www.freecodecamp.org/news/how-to-write-a-good-readme-file/
Add more relevant sections as specified in this link.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,96 @@
 # github-audit
 This service observes GitHub commits, operations and sends to an observability platform
```
Comment (upendrasahu) [inline README.md:2]: Add sample output metrics

File: README.md [hunk 2]
```diff
@@ -1,2 +1,96 @@
...
+
+
+# Build Instruction
+- Project uses golang 1.18 . 
+- Make sure golang environment is setup.
+- Simple run the Makefile to build the binary
+```bash
+    make build
+```
```
Comment (upendrasahu) [inline README.md:11]: have u added Dockerfile, and steps for docker build?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,96 @@
...
+- This will generate the github-audit binary named as ***github-audit***
+
+# Usage instruction
+- To view help menu 
+```bash
+./github-audit help
```
Comment (upendrasahu) [inline README.md:17]: Display all help options here

**Review by srivathsa-mugalodi-ML — APPROVED** [review]
Will approve for now to have basic Readme up on repository. Incorporate comments in subsequent pull request.

**Pull request merged**